commerce-order-content-web/src/main/resources/META-INF/resources/pending_orders/b2b/view.jsp

```diff
@@ -1,3 +1,5 @@
+<%@ include file="/init.jsp" %>
```

Comment:
mmmmm this should still be sorted where it was. Guessing the IDE may have done this on its own